chrome/app.js

```diff
@@ -62,8 +62,14 @@ GmailToTrello.App.prototype.bindEvents = function() {
         self.popupView.validateData();
     });
 
+    this.model.event.addListener('onLoadThreadTrelloCardsSuccess', function () {
+        self.popupView.buildPopupHtml(self.model.trello.threadCards);
+    });
```

Comment:
Loading the button now goes through this cycle where it requests Trello cards, then onLoadThreadTrelloCardsSuccess it builds the HTML and places the button on the page.